test-lib-openshift.sh

```diff
@@ -245,17 +249,36 @@ function ct_os_new_project() {
 # Deletes the specified project in the openshfit
 # Arguments: project - project name, uses the current project if omitted
 function ct_os_delete_project() {
+  if [ "${CT_SKIP_NEW_PROJECT:-false}" == 'true' ] ; then
+    echo "Deleting project skipped, cleaning objects only."
+    ct_delete_all_objects
+    return
+  fi
   local project_name="${1:-$(oc project -q)}" ; shift || :
   oc delete project "${project_name}"
 }
 
+# ct_delete_all_objects
+# -----------------
+# Deletes all objects within the project.
+# Handy when we have one project and want to run more tests.
+function ct_delete_all_objects() {
+  for x in bc builds is dc svc po routes secrets ; do
```

Comment:
Only these types? or everything? - see `oc types`

Reply:
Thanks, will extend this list.

Reply:
Current list is:
https://docs.openshift.com/container-platform/3.9/cli_reference/basic_cli_operations.html#object-types

Thanks for the `oc types` tip :).

Reply:
Actually I did not include events, since it produced too many errors that I'm not able to remove them (external cluster). I think it's ok to keep events as a garbage after the test run.